content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -292,3 +363,83 @@ try {
   span.setStatus({ code: otel.SpanStatusCode.ERROR });
 }
 ```
+
+## Using `sdk-trace-base` and manually propagating span context
+
+In some cases, you may not be able to use either the Node.js SDK nor the Web
+SDK. The biggest difference, aside from initialization code, is that you'll have
+to manually set spans as active in the current context to be able to create
+nested spans.
+
+### Initilizing tracing
+
+Initializing tracing is similar to how you'd do it with Node.js or the Web SDK.
+
+```javascript
+import opentelemetry = "@opentelemetry/api";
+import {
+  BasicTracerProvider,
+  ConsoleSpanExporter,
+  BatchSpanProcessor,
+} from "@opentelemetry/sdk-trace-base";
+
+const provider = new BasicTracerProvider();
+
+// Configure span processor to send spans to the exporter
+provider.addSpanProcessor(new BatchSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+// This is what we'll access in all instrumentation code
+export const tracer = opentelemetry.trace.getTracer(
+  'example-basic-tracer-node'
+);
+```
+
+Like the other examples in this document, this exports a tracer you can use
+throughout the app.
+
+## Creating nested spans
```

Comment:
It is not, I'll clarify in the title. Creating nested spans is harder in the non-Node/non-browser SDK case, because you need to manually set the current span in context before creating a nested span when using only `sdk-trace-base`. And we should definitely not make that the default recommendation, since it's the non-majority case and more code.

Reply:
Ah, makes sense, then the issue is the heading level, it probably should be `###`

Reply:
Thanks, missed this!